server/visualdl/mock/squeezenet_graph.json

```diff
@@ -0,0 +1,2072 @@
+{
+    "input": [
```

Comment:
this JSON file is too large, can we use a smaller model or just some pseudo-data?
just checking the logic do not need a full model.

If others need a complex JSON, give them offline seems more reasonable.